packages/web-components/src/components/link-list/link-list.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { html, customElement, property, LitElement } from 'lit-element';
+import settings from 'carbon-components/es/globals/js/settings';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
+import styles from './link-list.scss';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Link list type
+ */
+export enum LINK_LIST_TYPE {
+  /**
+   * Default
+   */
+  DEFAULT = 'default',
+
+  /**
+   * Vertical
+   */
+  VERTICAL = 'vertical',
+
+  /**
+   * Horizontal
+   */
+  HORIZONTAL = 'horizontal',
+
+  /**
+   * End of section
+   */
+  END = 'end',
+}
+
+/**
+ * Link list.
+ *
+ * @element dds-link-list
+ */
+@customElement(`${ddsPrefix}-link-list`)
+class DDSLinkList extends LitElement {
+  @property()
```

Comment:
Assuming we have an attribute selector for `type`. If not, please disregard:

```suggestion
  @property({ reflect: true })
```